client/cli/go/commands/tests/sample.json

```diff
@@ -1,100 +0,0 @@
-{
```

Comment:
I have this file in client/cli/go/cmds/tests/sample.json, Will add that with the PR for cli tests. I will add this file with this PR if it is block it. Can I know why is sample.json is required here?